x/exchange/spec/01_concepts.md

```diff
@@ -33,35 +36,41 @@ Fees can only be managed with a governance proposal using the [MsgGovManageFeesR
 
 Each market has a set of optional details designed for human-use, e.g. name, description, website url.
 
-A market is responsible (off-chain) for identifying order matches and triggering (on-chain) settlement.
+If a market accepts orders, it is responsible (off-chain) for identifying order matches and triggering (on-chain) settlement.
+If a market accepts commitments, it is able to manage committed funds and is responsible for releasing commitments as needed.
 
 A market receives fees for order creation and order settlement. It also defines what fees are required and what is acceptable as payments.
 
 A market can delegate various [permissions](#market-permissions) to other accounts, allowing those accounts to use specific endpoints on behalf of the market.
 
-Markets can restrict who can create orders with them by defining account attributes that are required to create orders. See [Required Attributes](#required-attributes).
+Markets can restrict who can create orders or commitments with them by defining account attributes that are required to create orders. See [Required Attributes](#required-attributes).
 
 Markets can control whether user-settlement is allowed.
 When user-settlement is allowed, the [FillBids](03_messages.md#fillbids) and [FillAsks](03_messages.md#fillasks) endpoints can be used for orders in the market.
 
 A market can also control whether orders can be created for it.
 When order creation is not allowed, any existing orders can still be settled or cancelled, but no new ones can be made (in that market).
 
+A market can separately control whether funds can be committed to it.
+When commitments are not allowed, any existing commitments can still be settled or cancelled, but no new funds can be committed (to that markert).
+
 The fees collected by a market are kept in the market's account, and can be accessed using the [MarketWithdraw](03_messages.md#marketwithdraw) endpoint.
 
 See also: [Market](03_messages.md#market).
 
+
 ### Required Attributes
 
-There is a separate list of attributes required to create each order type.
-If one or more attributes are required to create an order of a certain type, the order creator (buyer or seller) must have all of them on their account.
+There is a separate list of attributes required to create each order type and commitments.
+If one or more attributes are required to create something, creator (buyer, seller, or committer) must have all of them on their account.
```

Comment:
There is an issue with parallelism in this sentence.

If one or more attributes are required to create something, `the` creator (buyer, seller, or committer) must have all of them on their account.

Reply:
It's not entirely clear what `something` is, and it's better to use precise wording in documentation.

Reply:
The first sentence says what actions might have required attributes. The second sentence tells you how they behave.

I rephrased the second to this:
> If one or more attributes are required for an action, the associated account (e.g. buyer, seller, or committer) must have all of them on their account.

That's as precise as it should be. In that sentence, it doesn't matter what the exact action the required attributes are related to since it's describing the general concept of required attributes.

Reply:
This was referring to the second sentence. Thanks for fixing it!